dockers/docker-sonic-telemetry/base_image_files/monit_telemetry

```diff
@@ -2,4 +2,4 @@
 ## Monit configuration for telemetry container
 ###############################################################################
 check program container_memory_telemetry with path "/usr/bin/memory_checker telemetry 419430400"
-    if status == 3 for 10 times within 20 cycles then exec "/usr/bin/restart_service telemetry"
+    if status == 3 for 10 times within 20 cycles then exec "/usr/bin/restart_service telemetry" repeat every 2 cycles
```

Comment:
>exec [](http://example.com/codeflow?start=54&length=4)

Please explain how do you test this PR? The test could be either manual or ideally automated. You need to make sure the old code fails the test and new code passes the test.

Reply:
As we discussed at last week's meeting, I am working on the PR of  pytest testcase to make sure the old code will fail while new one will succeed.

Reply:
Pytest PR is submitted for review: https://github.com/Azure/sonic-mgmt/pull/5492.